### PR TITLE
ui bug fixes

### PR DESCRIPTION
**Pull request by AdithyaShaj**
Fixing Loader height
Calendar hides on date selection - fixed
DC Open Orders - fixed alignment issues in cards
Fixed style issues for home page cards

### CODE REVIEW

File: src/components/homePage.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ const CardTitle = ({
   classname?: string;
 }) => {
   return (
-    <div className={`${classname} flex justify-between`}>
+    <div className={`${classname ? classname : ""} flex justify-between`}>
```
Comment (thasnim-cm) [inline src/components/homePage.tsx]: we can add default value as empty string for classname prop

File: src/components/homePage.tsx [hunk 2]
```diff
@@ -236,9 +236,9 @@ const HomePage = () => {
             </div>
           </div>
           {isLoading ? (
-            <Loader />
+            <Loader className={"card-loader-height"} />
```
Comment (thasnim-cm) [inline src/components/homePage.tsx]: curly brackets can be removed

**Review by sarath-sasi-panickaveetil — APPROVED** [review]

File: src/index.css [hunk 3]
```diff
@@ -924,6 +925,15 @@ hr.horizontalLine.blue {
 ::-webkit-scrollbar {
   @apply w-1 h-1;
 }
+.card-loader-height {
+  @apply min-h-[7.188rem] sm:min-h-[17rem] lg:min-h-[16rem] xl:min-h-[9.5rem] 3xl:min-h-[8.5rem];
+}
+.sessions-loader-height {
+  @apply min-h-[24rem];
```
Comment (raj-kumar-litmus) [inline src/index.css]: for a single class why did u have to create a unique class name.
Use it directly.

File: src/index.css [hunk 4]
```diff
@@ -924,6 +925,15 @@ hr.horizontalLine.blue {
...
+}
+.opm-loader-height {
+  @apply min-h-[29rem];
```
Comment (raj-kumar-litmus) [inline src/index.css]: Same comment as above.

Comment (AdithyaShaj) [reply src/index.css]: this is used in multiple places. do we need to remove the class?

Comment (raj-kumar-litmus) [reply src/index.css]: yes. use it directly

**Pull request merged**